api/DocumentOrShadowRoot.json

```diff
@@ -5,40 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot",
         "support": {
           "chrome": {
-            "version_added": "53"
+            "version_added": "1"
```

Comment:
FWIW I think support statements for the mixin itself make no sense whatsoever. While the members of a mixin could be interpreted as "support on all the interfaces that include this mixin", support for the mixin itself doesn't really mean anything. But oh well, let's get rid of mixins another day.